public/content/history/index.md

```diff
@@ -188,6 +188,24 @@ The London upgrade introduced [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559
 
 </ExpandableCard>
 
+<ExpandableCard title="London Upgrade" contentPreview="What was the London Upgrade / EIP-1559?">
```

Comment:
Hesitant on just dropping this here since we don't have expandable cards like this for any of the other upgrades; they've been reserved for the EIP summaries.

This feels like it would make more sense to just be part of the "Summary" copy, trimming down any redundant info.